cmd/uplink/cmd/root.go

```diff
@@ -26,11 +30,15 @@ type UplinkFlags struct {
 
 var cfg UplinkFlags
 
+var debugPprof = flag.Bool("debug.pprof", false, "if true, creates cpu.prof and memory.prof results in current directory")
```

Comment:
Instead of making it a boolean, make it a string one for cpu and one for memory. This is the standard solution. https://golang.org/pkg/runtime/pprof/

```
-debug.cpuprofile cpu.prof -debug.memprofile mem.prof
```

I'm also wondering, whether it should be `-profile.cpu`, `-profile.mem` or similar, rather than just `debug`.